### DISCUSSION TITLE
Gitlab as IDP for unleash authentication

### DISCUSSION THREAD

**Discussion by kabilanmani93** [Unleash/unleash#6719] [Q&A]
Hi is it possible to use gitlab as IDP for unleash authentication ?

**ref**: https://docs.getunleash.io/reference/sso

**Comment by daveleek** [comment]
Hello @kabilanmani93 and thank you for your question.
We've not tested SSO with GitLab, so this response is based on what we see from their documentation found here: https://docs.gitlab.com/ee/integration/openid_connect_provider.html

From what I can tell, you'll have to use OIDC and that should work. But since group info is only provided in the userinfo endpoint, groups syncing to Unleash isn't going to work.

**Reply by kabilanmani93** [reply]
@daveleek we tried the gitlab sso itself is not working
![image](https://github.com/Unleash/unleash/assets/46244764/568f4ede-8e61-4353-bb62-d8d3344a4e5c)

**Reply by sighphyre** [reply]
Hey @kabilanmani93, that appears to be incorrect scopes set here but I suspect we've actually already spoken over Slack. DId you get what you needed?
